scorescreen.py

```diff
@@ -104,26 +103,52 @@ def run(self, game, scores):
 
             mos_x, mos_y = pygame.mouse.get_pos()
             game.screen.fill(BLACK)
-            game.screen.blit(self.scorescreen, (350, 0))
-
-            msg = self.font3.render(_("GAME OVER"), 2, WHITE)
-            game.screen.blit(msg, (350, 120))
-            scoress = self.font2.render(_("SCORE      ") +
-                                        _(str(scores)), 2, WHITE)
-            game.screen.blit(scoress, (500, 265))
-            scoress = self.font2.render(_("BEST        ") +
-                                        _(str(max_score)), 2, WHITE)
-            game.screen.blit(scoress, (510, 330))
-
-            if self.restart.get_rect(
-                    center=(550 + 60, 420 + 60)).collidepoint(mos_x, mos_y):
-                game.screen.blit(pygame.transform.scale(
-                    self.restart, (124, 124)), (550 - 2, 420 - 2))
+            game.screen.blit(self.scorescreen, (game.background_x, 0))
+
+            # Display GAME OVER in the middle of screen
```

Comment:
Redundant comment, please remove.

Also remove other redundant comments .ie. comments where the code makes it obvious what's being achieved.